### PR TITLE
Spectra: Update to v0.0.10

### PR DESCRIPTION
**Pull request by ewels**

### CODE REVIEW

**Comment by epruesse** [comment]
I've restarted the AppVeyor build -- the failures were claiming the bat file was missing, so potentially just a build system issue. Travis seems to be just way backlogged on the macos builds.

**Comment by ewels** [comment]
Thanks @epruesse! Any idea why my earlier commits with the versions specified failed? It shouldn’t matter I think, just curious what I did wrong..

**Comment by epruesse** [comment]
@ewels Whitespace. It needs to be `- package versionspec` so `networkx =2.0`. There also can't be a whitespace between the comparison operator and the value, so no `- networkx >= 2`.

**Comment by epruesse** [comment]
@ewels The failures on AppVeyor for py2.7 and py3.6 didn't go away. Maybe ask on gitter? Or try a manual rerender of the feedstock?

**Comment by ewels** [comment]
Yeah, I had the same problem with https://github.com/conda-forge/colormath-feedstock/pull/2 and I just merged anyway. The docs all seem to say that there's no need for a `bat` file but then it fails for that reason, which makes no sense for me.

Phil

**Comment by epruesse** [comment]
Try to rerender the feedstock. Maybe the scripts need to be updated. Looking at the history for conda-forge/staged-recipes, the problem doesn't appear to exist there. If that fails, ping core...

File: recipe/meta.yaml [hunk 1]
```diff
@@ -22,6 +22,7 @@ requirements:
 
   run:
     - python
+    - networkx
     - colormath
```
Comment (mbargull) [inline recipe/meta.yaml]: This needs
```yaml
  - networkx ==2.0
  - colormath ==3.0.0
```
due to
https://github.com/jsvine/spectra/blob/v0.0.10/requirements.txt
https://github.com/jsvine/spectra/blob/v0.0.10/setup.py#L2
https://github.com/jsvine/spectra/blob/v0.0.10/setup.py#L9-L10
https://github.com/jsvine/spectra/blob/v0.0.10/setup.py#L40

**Review by mbargull — CHANGES_REQUESTED** [review]
Dependencies need to be version-constrained, matching `setuptools`' `install_requires` (otherwise you'll need to patch `requirements.txt` or `setup.py` to modify those).
Also, this package could be build with `noarch: python`, i.e.,
```yaml
build:
  number: 0
  noarch: python
  script: python setup.py install --single-version-externally-managed --record record.txt
```
. That way you needn't care about AppVeyor.
But either way you'll need to rerender the feedstock.

**Comment by ewels** [comment]
@conda-forge-admin, please rerender

**Comment by mbargull** [comment]
Yeah, just like a pro, @ewels! Good job!

**Comment by ewels** [comment]
Haha, thanks for the tuition @mbargull 😉

**Pull request merged**